app/styles/app.scss

```diff
@@ -129,3 +147,7 @@ a.sub-navigation:hover {
 .app-content {
     padding-left: 40px;
 }
+
+.ace-editor {
```

Comment:
Overall this looks great. 

As a note, I'm moving this to app/styles/components/ace-editor.scss and importing at the top of this file before merge.